components/user/UserSignIn.vue

```diff
@@ -47,14 +47,6 @@ async function oauth() {
   }
 }
 
-async function handleInput() {
```

Comment:
I had to move the `handleInput` definition below `filteredServers` to fix an eslint "used before defined" error, I think this was a false positive tho.